app/src/main/cpp/skyline/gpu/native_window.h

```diff
@@ -0,0 +1,303 @@
+// SPDX-License-Identifier: MPL-2.0
+// Copyright © 2021 Skyline Team and Contributors (https://github.com/skyline-emu/)
+// Copyright © 2021 The Android Open Source Project
+
+#pragma once
+
```

Comment:
We're not having file comments, they're inconsistent and weird to have because of how blatantly simple file-based categorization generally ends up being like you could easily collect that information in the time.. I didn't realize this was the native window file I agree now but I took a lot of time to write up this comment so I will post it regardless so you have an equivalent amount of time wasted by reading it.

Reply:
😆